### ISSUE TITLE
Need Way To Get Multiple Media Objects on Content

### ISSUE THREAD

**Issue by volomike** [flextype/flextype#481]
I wanted to list out images from the CMS without knowing their names head of time. I needed to collect them together somehow, and then use an API to list out that collection.

I attempted:

Attempt 1: I created an entry like a folder and then put one single default content in there with an attached media object. I was unable to use /api/files, /api/folders, /api/entries, or /api/images to get the list of images for a given piece of content.

Attempt 2: I created a single entry and then, off itself, I created multiple media objects. Again, I was unable to use the API to get a wildcard of the images.

**Comment by volomike** [comment]
Reading the source code, I found the solution, and think that you have an error in the docs under Fetch Files Collection.

`GET /api/files?path=entries/gallery&token={YOUR_FILES_TOKEN}`

I believe that it's not `id` that it wants as a parameter, but `path`.

**Comment by Awilum** [comment]
yes, seems it is a mistake in the documentation. `path` should be instead of `id`
https://docs.flextype.org/en/rest-api/files
